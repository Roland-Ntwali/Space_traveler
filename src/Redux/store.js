import { configureStore } from '@reduxjs/toolkit';
import rocketReducer from './Rocket/rocketSlice';
import missionSlice from './Mission/missionSlice';

const store = configureStore({
  reducer: {
    rocketReducer,
    missions: missionSlice,
  },
});

export default store;
