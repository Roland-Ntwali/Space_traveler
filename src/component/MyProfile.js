/* eslint-disable */
import React from "react";
import UpdateRocket from "./RocketProfile";
import Missionprofile from "./missionProfile";


const MyProfile = () => {

  return(
    <table className="profile_d">
      <Missionprofile />
      <UpdateRocket />
    </table>
  )
};

export default MyProfile;
