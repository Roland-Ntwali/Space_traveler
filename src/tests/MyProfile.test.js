import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import MyProfile from '../component/MyProfile';
import store from '../Redux/store';
/* eslint-disable */
jest.setTimeout(10000);
describe("My profile", () => {
  it("it has to Render a list", () => {
    const profile = render(
      <>
        <Provider store={store}>
          <MyProfile />
        </Provider>
      </>
    );
    expect(profile).toMatchSnapshot();
  });
});
