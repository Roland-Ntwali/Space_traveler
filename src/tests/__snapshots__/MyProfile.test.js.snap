// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My profile it has to Render a list 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <table
        class="profile_d"
      >
        <div>
          <h1
            class="t_head"
          >
            My Missions
          </h1>
          <table>
            <tbody />
          </table>
        </div>
        <div>
          <h1
            class="t_head"
          >
            My Rockets
          </h1>
          <table
            class="table"
          >
            <tbody
              class="t_body"
            />
          </table>
        </div>
      </table>
    </div>
  </body>,
  "container": <div>
    <table
      class="profile_d"
    >
      <div>
        <h1
          class="t_head"
        >
          My Missions
        </h1>
        <table>
          <tbody />
        </table>
      </div>
      <div>
        <h1
          class="t_head"
        >
          My Rockets
        </h1>
        <table
          class="table"
        >
          <tbody
            class="t_body"
          />
        </table>
      </div>
    </table>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
